services/tracks/src/services/genresService.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+const { v4: uuid } = require('uuid');
+const { Genres } = require('../db/models/Genres');
+const { errorMessages } = require('../config');
+
+class GenreService {
+  async getGenres() {
+    const genres = await Genres.findAll({ order: ['id'] });
+    return genres;
+  }
+
+  async getGenreById(id) {
+    const genre = await Genres.findOne({ where: { id } });
+
+    if (!genre) throw new Error(errorMessages.GENRE_NOT_EXISTS_ID);
+
+    return genre;
+  }
+
+  async createGenre(genreData) {
+    const genreID = uuid();
+    genreData.id = genreID;
+
+    const createdGenre = await Genres.create(genreData);
+
+    return createdGenre;
```

Comment:
I think you should return it immediately. In any case, the asynchronous function will return a promise, so there is no need to wait for the result of the service Genres
```suggestion
    return Genres.create(genreData);
```